<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraControlWidgetClass</class>
 <widget class="QWidget" name="CameraControlWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>472</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="widget_camera" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1" colspan="4">
       <widget class="QComboBox" name="binningModeBox">
        <property name="editable">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>AVERAGE</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Binning:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>binning mode:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="5">
       <widget class="QPushButton" name="refreshBtn">
        <property name="text">
         <string>Refresh</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>binning:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="2">
       <widget class="QLabel" name="etSta">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Binning Mode:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLabel" name="binningSta">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QLabel" name="binningmodeSta">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>exposure time:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="4">
       <widget class="QComboBox" name="binningNumBox">
        <property name="editable">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>ONE_BY_ONE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TWO_BY_TWO</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0" colspan="5">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Exposure</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Exposure Time:(ms)</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="3">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Initial Exposure Time:(ms)</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLineEdit" name="initEtLint"/>
         </item>
         <item row="1" column="4">
          <widget class="QPushButton" name="autoExpBtn">
           <property name="text">
            <string>AutoExposure</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>GrayLevel:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3" colspan="2">
          <widget class="QLineEdit" name="EtText"/>
         </item>
         <item row="2" column="3" colspan="2">
          <widget class="QProgressBar" name="graylevleBar">
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>format type:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="2">
       <widget class="QLabel" name="format">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>259</width>
       <height>122</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
